reverse_linkedlist/jade_solution.py

```diff
@@ -0,0 +1,41 @@
+def get_data_list(data):
+  data_list = []
+  data_list.append(one['data'])
```

Comment:
You can initialize `inner_dict` to `one`, and simply start the loop
```suggestion
  inner_dict = one
  while inner_dict is not None:
```